build/checks/images.py

```diff
@@ -0,0 +1,41 @@
+# coding=utf8
+# the above tag defines encoding for this document and is for Python 2.x compatibility
+
+import sys
+import os
+import re
+import requests
```

Comment:
Apparently, this isn't a module. Are you sure it should be imported? It's causing issues.